web-common/src/features/dashboards/pivot/PivotTable.svelte

```diff
@@ -124,26 +137,83 @@
       }
     }
   };
+
+  function calculateFirstColumnWidth(firstColumnName: string) {
+    const rows = $pivotDataStore.data;
+
+    const isTimeDimension = firstColumnName.startsWith("__time");
```

Comment:
`__time` is not the default column name. Some of our dashboard usually have that as the key but in practice it could be anything. You would want to use `isTimeDimension` function here